src/environment/conda_prefix.rs

```diff
@@ -192,7 +192,9 @@ impl CondaPrefixUpdater {
                     tokio::task::spawn_blocking(move || prefix_clone.find_installed_packages())
                         .unwrap_or_else(|e| match e.try_into_panic() {
                             Ok(panic) => std::panic::resume_unwind(panic),
-                            Err(_err) => Err(miette::miette!("the operation was cancelled")),
+                            Err(e) => {
+                                std::panic::resume_unwind(Box::new(PrefixError::JoinError(e)))
```

Comment:
I don't get why you'd unwind here, if you just checked that this code path didn't panic

Reply:
Removed it, and added a test